installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,58 @@
+[id="preparing-to-install-on-vsphere"]
+= Preparing to install on vSphere
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-vsphere
+
+toc::[]
+
+
+[id="preparing-to-install-on-vsphere-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+* If you use a firewall and plan to use Telemetry, you
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] required by your cluster.
+
+
+[id="choosing-a-method-to-install-ocp-on-vsphere"]
+== Choosing a method to install {product-title} on vSphere
+
+You can install {product-title} on vSphere by using installer-provisioned or user-provisioned infrastructure. The default installation type uses installer-provisioned infrastructure, where the installation program provisions the underlying infrastructure for the cluster. You can also install OpenShift Container Platform on infrastructure that you provide. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
```

Comment:
s/OpenShift Container Platform/{product-title}

Reply:
Done, thank you!